install/kubernetes/cilium/charts/hubble-ui/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+You have successfully installed Cilium {{ .Chart.Name }}
```

Comment:
maybe mention it's the Hubble UI that has been installed here?

Reply:
It will pull the respective chart value. So here it should pull the 'hubble-u'.
The final output would be 'Cilium hubble-ui'